lineapy/transformer/node_transformer.py

```diff
@@ -739,3 +764,49 @@ def get_source(self, node: ast.AST) -> Optional[SourceLocation]:
             end_lineno=node.end_lineno,  # type: ignore
             end_col_offset=node.end_col_offset,  # type: ignore
         )
+
+    def get_else_source(self, node: ast.If) -> Optional[LiteralNode]:
+        body_source = self.get_source(node.body[-1])
+        orelse_source = (
+            self.get_source(node.orelse[0]) if node.orelse else None
+        )
+        assert body_source, "Body of If/Else must have at least one statement"
+        if not orelse_source:
+            return None  # If there is no else block, the else keyword would not be present
+        return self.tracer.literal(
+            "else:",
+            SourceLocation(
+                source_code=self.source_code,
+                lineno=body_source.end_lineno
+                + 1,  # Else node can only start one line after if block
+                col_offset=0,
+                end_lineno=max(
+                    orelse_source.lineno - 1, body_source.end_lineno
+                ),  # Keyword else can be in the previous line or the same line as the first statement of the else block
+                end_col_offset=orelse_source.col_offset,
+            ),
+        )
+
+    def get_black_box_without_executing(
+        self, nodes: List[ast.stmt]
```

Comment:
use ast.any (might be ast.Any - there should be examples)

Reply:
its ast.AST